uvloop/handles/stream.pyx

```diff
@@ -1,4 +1,8 @@
-DEF __PREALLOCED_BUFS = 4
+cdef extern from *:
+    '''
+    enum {__PREALLOCED_BUFS = 4};
+    '''
+    const bint __PREALLOCED_BUFS
```

Comment:
Am curious whether it would make sense to make this a Cython `enum` (or if there was some issue seen doing that)

```suggestion
cdef enum:
    __PREALLOCED_BUFS = 4
```

Reply:
Yeah, I think this is a good suggestion - it pretty much generates the same C code.

Reply:
Thanks Fantix! 🙏

Submitted a PR: https://github.com/MagicStack/uvloop/pull/634